### ISSUE TITLE
Enable easy inspection of MARC for an Alma record

### ISSUE THREAD

**Issue by acka47** [hbz/lobid-resources#1316]
Going forward with improving lobid-alma and setting up Alma-based NWBib, we will have the recurring need to inspect Alma MARC records based on an entry in alma.lobid.-org/resources. There are two options to implement this:
 
1. Add a simple HTML view for each entry (e.g. at https://alma.lobid.org/resources/99370878737906441) that links to the MARC an the JSON for comparison.
2. Add a way to add a format extenstion like `.mrc` to a URL to get the MARC record, e.g. https://alma.lobid.org/resources/99370878737906441.mrc, similar to https://alma.lobid.org/resources/99370878737906441.json which is already working.

**Comment by TobiasNx** [comment]
Wasn*t there a problem, that the MARC data is not open yet?

**Comment by dr0i** [comment]
Right - we are waiting on an official decision by the Verbundgruppe to expose the full data. @acka47 notied that the underlying MAB data was already released as CC0. However, we could you the internal API to get the MARC data and expose it publicly via our API as single pages.
@blackwinter do you have the link to the wiki-page where we noted the task of questioning Verbundgruppe? How shall we proceed further?

**Comment by blackwinter** [comment]
> do you have the link to the wiki-page where we noted the task of questioning Verbundgruppe?

You mean [NZ-Publishing - Fragen an Verbundgruppe](https://service-wiki.hbz-nrw.de/pages/viewpage.action?pageId=613646369)?

> How shall we proceed further?

I have no idea, sorry.

**Comment by dr0i** [comment]
Thx @blackwinter ! Maybe @acka47 can solve ask the question at Verbundgruppe with you in CC so we can open the data if they give their ok.

**Comment by acka47** [comment]
AFAIK, @trugwaldsaenger is checking the legal status of the Alma-based data. @trugwaldsaenger, please let us know as soon as you know more.

**Comment by dr0i** [comment]
Verbundgruppe decided that any local (IZ) field is considered to not be open data. We [have filtered these out](  https://github.com/hbz/lobid-resources/issues/1687) thus everything in lobid-resources is Open Data again.

Re this issue:
we've already enabled viewing the MARC-XML source, but a little bit hidden - hover the mouse pointer a bit left of the RDF symbol (e.g. https://lobid.org/resources/99370878737906441). An `<M>` symbol appears which is linked to the MARC-XML , e.g. https://alma.lobid.org/marcxml/99370878737906441.

So:
1. we should "unhide" the MARC-XML download link
2. we might consider to provide easier usage by suffixing `.mrc` to the resource

**Comment by acka47** [comment]
> 1. we should "unhide" the MARC-XML download link
> 2. we might consider to provide easier usage by suffixing `.mrc` to the resource

+1 for both

**Comment by dr0i** [comment]
functional review: @acka47 
code review: @fsteeg

**Comment by dr0i** [comment]
@acka47  you can test using "stage", e.g.  http://stage.lobid.org/resources/99370672908406441

**Comment by acka47** [comment]
The link is now unhidden but `.mrc` does not work for me: http://stage.lobid.org/resources/99370672908406441.mrc

**Comment by dr0i** [comment]
> 2. Add a way to add a format extenstion like .mrc to a URL to get the MARC record,

No, file extension for MARC-XML ist `mrcx`, see https://www.wikidata.org/wiki/Q28770433.
http://stage.lobid.org/resources/99370672908406441.mrcx

**Comment by acka47** [comment]
Yes`.mrcx` works. Thanks! +1

**Comment by dr0i** [comment]
Deployed - closing.
